express/scripts/utils.js

```diff
@@ -106,14 +106,28 @@ export function removeIrrelevantSections(area) {
 }
 
 function overrideMiloColumns(area) {
-  if (!area) return;
-  area.querySelectorAll('main > div').forEach((section) => {
+  const replaceColumnBlock = (section) => {
     const columnBlock = section.querySelectorAll('div.columns');
     columnBlock.forEach((column) => {
       if (column.classList[0] !== 'columns') return;
       column.classList.remove('columns');
       column.className = `ax-columns ${column.className}`;
     });
+  };
+
+  const repalceTableOfContentBlock = (section) => {
```

Comment:
what if I've authored a table-of-contents block coming from milo? 
We only want to replace our old existing table of contents. It should be possible to author a milo table-of-contents or ax-table-of-contents though.